internal/cri/server/helpers.go

```diff
@@ -141,6 +143,11 @@ func (c *criService) getContainerRootDir(id string) string {
 	return filepath.Join(c.config.RootDir, containersDir, id)
 }
 
+// getImageVolumeHostPath returns the image volume directory for share.
+func (c *criService) getImageVolumeHostPath(id string) string {
+	return filepath.Join(c.config.RootDir, imageVolumeDir, id)
```

Comment:
Suggestion: use `c.config.StateDir` because image volume is temporary mount and gone after VM restart.